pkg/operator/controller/machineconfig/constants.go

```diff
@@ -96,6 +96,11 @@ const (
 )
 
 var (
+	// CrioUnixSocketConfData contains the configuration required
+	// for enabling CRI-O profiling
+	CrioUnixSocketConfData = fmt.Sprintf(`[Service]
+Environment="%s"`, CrioUnixSocketEnvString)
+
```

Comment:
This is already in `main` from another PR. I suppose it's due to some unlucky rebase.